.editorconfig

```diff
@@ -5,6 +5,12 @@ end_of_line = lf
 indent_style = space
 indent_size = 2
 charset = utf-8
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[CMakeLists.txt]
+indent_style = tabs
```

Comment:
Any reason to use tabs here where space is used in the source files?

Reply:
Hi Martin,

the original file was inconsistent. If you wanted to say "Hey Manuel, I kept some inconsistent stuff in the repo and it would be great if you could fix it. Thanks. Bye" Then I can fix it and revert my changes of the file. 

My usual mode is to preserve style in the original files and help to keep it. So, I changed the editorconfig configuration. Next time I'll ask before making such changes.

Do you have other input on this initial TravisCI integration? Things that need to be in that you can approve the changes?

Best
Manuel